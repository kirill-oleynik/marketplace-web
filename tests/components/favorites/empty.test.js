const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const { EmptyFavorites } = require('../../../app/components/favorites/empty');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <EmptyFavorites
        t={(translation) => translation}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
