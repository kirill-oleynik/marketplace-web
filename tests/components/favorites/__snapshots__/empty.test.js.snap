// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div
  className="message-block"
>
  <img
    alt=""
    className="message-block__img message-block__img--favorites"
    src="/static/images/favorites.png"
    srcSet="/static/images/favorites@2x.png 2x, /static/images/favorites@3x.png 3x"
  />
  <p
    className="font-24 mb-10"
  >
    empty.title
  </p>
  <p
    className="font-14 mb-30"
  >
    empty.description
  </p>
  <Link
    href="/"
  >
    <MainButton
      className=""
      color="blue"
      size="md"
      type="button"
    >
      empty.backToHome
    </MainButton>
  </Link>
</div>
`;
